nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestSQLRepositoryProperties.java

```diff
@@ -1269,12 +1271,12 @@ protected void changeDoc(String id, StateDiff diff) throws Exception {
     public void testComplexListArrayElementResetWithNull() {
         DocumentModel doc = session.createDocumentModel("/", "mydoc", "MyDocType");
         doc.setPropertyValue("complexlist",
-                (Serializable) Arrays.asList(Collections.singletonMap("array", new String[] { "foo" })));
+                (Serializable) Collections.singletonList(Collections.singletonMap("array", new String[]{"foo"})));
```

Comment:
I don't see this as a cleanup, it's longer. And in the future we'll move to Java 9's `List.of` anyway...
Also in our formatting the string array needs spaces on both sides of the braces, like it was before.